### ISSUE TITLE
notebook error in modelling step

### ISSUE THREAD

**Issue by xvazquezc** [aqlaboratory/openfold#122]
Another error, this in the `Run OpenFold and download prediction` section

```
---------------------------------------------------------------------------

UnpicklingError                           Traceback (most recent call last)

[<ipython-input-6-28899244d813>](https://localhost:8080/#) in <module>()
     51         of_model_name
     52       )
---> 53       d = torch.load(params_name)
     54       openfold_model.load_state_dict(d)
     55     else:

1 frames

[/usr/local/lib/python3.7/dist-packages/torch/serialization.py](https://localhost:8080/#) in _legacy_load(f, map_location, pickle_module, **pickle_load_args)
    918             "functionality.")
    919 
--> 920     magic_number = pickle_module.load(f, **pickle_load_args)
    921     if magic_number != MAGIC_NUMBER:
    922         raise RuntimeError("Invalid magic number; corrupt file?")

UnpicklingError: invalid load key, 'v'.
```

**Comment by SauravMaheshkar** [comment]
+1. I also ran into this error.

**Comment by gahdritz** [comment]
It was an issue with git-lfs, which apparently isn't installed by default on Colab. Fixed in c401749.
